### ISSUE TITLE
An error in AttentionSeq2seq

### ISSUE THREAD

**Issue by qqqulu** [farizrahman4u/seq2seq#77]
I tried to use AttentionSeq2seq(depth >1), but i always got an error:
**Error when checking model input: expected layer_1 to have shape (None, 150, 256) but got array with shape (640L, 150L, 205L)**
my input dim = 205, input length is 150, hidden dim is 256.
why do i getthis error? am I mistaken@@? 

I found there is a Layer added in AttentionSeq2seq model:
**self.add(Layer(batch_input_shape=shape if depth[0] == 1 else shape[:2] + (hidden_dim,)))**
so,depth == 1, shape is (n_samples, input_length, input_dim)
depth >1, shape is (n_samples, input_length, hidden_dim)
the problem is caused by this? and why we need to add this layer

**Comment by wobu** [comment]
Should be fixed with the last commit 123a2cfe9e5c4db41856eaff573b9d08cdb534dc

Try to use the master via:

`pip uninstall seq2seq`
`pip install git+https://github.com/fchollet/keras.git`
